cmake/TaichiExportCore.cmake

```diff
@@ -0,0 +1,6 @@
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Why does this require v3.0 while examples/chi_examples/CMakeLists.txt requires v3.12?